languages/elixir/exercises/concept/basic-strings/.docs/instructions.md

```diff
@@ -0,0 +1,67 @@
+In this exercise, you are going to help high school sweethearts profess their love on social media by generating an ASCII heart with their initials:
```

Comment:
I love (pun intended) the theme! It is simple and suits the purpose of learning how to work with strings very well!